addons/udes_stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -877,6 +889,16 @@ def _create_batch(self, user_id, picking_type_id, picking_priorities=None, picki
         The batch will be marked as ephemeral.
         In case no pickings exist, return None.
         """
+        # TODO - This function actually searches for one picking but it talks about pickings... why?
+        #        What is this function actually supposed to do?
+        #        It seems that it should search and add multiple pickings since we search by picking type
+        #        But actually it only assigns the first picking found. This is prone to issues if we create 2
+        #        pickings and then call this function as potentially either could be selected first. This was
```

Comment:
Which one is selected first relies on the `_order` defined at stock.picking model, there is a note about it in `search_for_pickings`.

Reply:
Thanks, that field has changed between Odoo 11 and Odoo14 which explains why the difference in behaviour in the tests.